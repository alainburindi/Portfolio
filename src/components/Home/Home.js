import React from 'react';
import { Link } from 'react-router-dom';

const profileImage = 'https://res.cloudinary.com/al-tech/image/upload/v1568369434/ahrhivsypsuedoc8lzbn.jpg';
const Home = () => (
  <div className="hero is-fullheight">
    <div className="hero-body">
      <div className="container v-centered">
        <div className="columns">
          <div className="column has-text-centered">
            <img src={profileImage} alt="Alain" className="profile-image" />
          </div>
          <div className="column">
            <h1 className="title is-1">I am Alain Burindi</h1>
            <h3 className="title is-5">Software Engineer -2 Years</h3>
            <hr />

            <p className="about-me">A backend developer, who love building projects</p>
            <div className="columns">
              <div className="column">
                <Link to="/resume" className="button is-info is-large is-fullwidth">
                  RESUME
                </Link>
              </div>
              <div className="column">
                <Link to="/contact" className="button is-info is-large is-fullwidth">
                  CONTACT
                </Link>
              </div>
            </div>
            <p className="social-icons">
              <a
                href="https://github.com/alainburindi"
                target="_blank"
                className="social-icon"
                rel="noopener noreferrer"
              >
                <i className="fab fa-github" />
              </a>
              <a
                href="https://twitter.com/alainburindi"
                target="_blank"
                className="social-icon"
                rel="noopener noreferrer"
              >
                <i className="fab fa-twitter" />
              </a>
              <a
                href="https://facebook.com/alainburindi"
                target="_blank"
                className="social-icon"
                rel="noopener noreferrer"
              >
                <i className="fab fa-facebook" />
              </a>
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
);

export default Home;
