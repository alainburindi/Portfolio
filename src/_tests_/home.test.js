import React from 'react';
import Home from '../App/components/Home/Home';
import { shallow } from '../enzyme';

test('should render the home page', () => {
  const home = shallow(<Home />);
  expect(home).toMatchSnapshot();
});
