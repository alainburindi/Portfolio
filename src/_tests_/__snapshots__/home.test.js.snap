// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the home page 1`] = `ShallowWrapper {}`;
